files/KoreBuild/config/runtime.version

```diff
@@ -1 +1 @@
-2.0.3
```

Comment:
You should only change this if you also re-target all build tools to 2.1.

The repos themselves take care of installing the runtime via the `<DotNetRuntime>` item group. example:  https://github.com/aspnet/Mvc/blob/dev/build/repo.props#L15-L18